pkg/remote/github/init.go

```diff
@@ -35,11 +35,16 @@ func Init(version string, alerter *alerter.Alerter,
 	deserializer := resource.NewDeserializer(factory)
 	providerLibrary.AddProvider(terraform.GITHUB, provider)
 
-	supplierLibrary.AddSupplier(NewGithubRepositorySupplier(provider, repository, deserializer))
-	supplierLibrary.AddSupplier(NewGithubTeamSupplier(provider, repository, deserializer))
-	supplierLibrary.AddSupplier(NewGithubMembershipSupplier(provider, repository, deserializer))
-	supplierLibrary.AddSupplier(NewGithubTeamMembershipSupplier(provider, repository, deserializer))
-	supplierLibrary.AddSupplier(NewGithubBranchProtectionSupplier(provider, repository, deserializer))
+	var githubSuppliers []resource.SimpleTypeSupplier
+	githubSuppliers = append(githubSuppliers, NewGithubRepositorySupplier(provider, repository, deserializer))
+	githubSuppliers = append(githubSuppliers, NewGithubTeamSupplier(provider, repository, deserializer))
+	githubSuppliers = append(githubSuppliers, NewGithubMembershipSupplier(provider, repository, deserializer))
+	githubSuppliers = append(githubSuppliers, NewGithubTeamMembershipSupplier(provider, repository, deserializer))
+	githubSuppliers = append(githubSuppliers, NewGithubBranchProtectionSupplier(provider, repository, deserializer))
+
+	for _, supplier := range githubSuppliers {
```

Comment:
Why not using `supplierLibrary.Suppliers()` to retrieve the list of suppliers instead of building a slice ?

Reply:
It will be filtered in next PR before being added to supplierLibrary

Reply:
Do you mean that the filtering will work as follow ?

If I only want to scan s3 buckets, only the s3 supplier will be inside the supplier library ?